core/src/main/java/io/questdb/griffin/engine/functions/groupby/StdDevPopGroupByFunctionFactory.java

```diff
@@ -46,4 +49,60 @@ public boolean isGroupBy() {
     public Function newInstance(int position, ObjList<Function> args, IntList argPositions, CairoConfiguration configuration, SqlExecutionContext sqlExecutionContext) {
         return new StdDevPopGroupByFunction(args.getQuick(0));
     }
+
+    private static class StdDevPopGroupByFunction extends AbstractStdDevGroupByFunction {
+
+        public StdDevPopGroupByFunction(@NotNull Function arg) {
+            super(arg);
+        }
+
+        @Override
+        public double getDouble(Record rec) {
+            long count = rec.getLong(valueIndex + 2);
+            if (count > 0) {
+                double sum = rec.getDouble(valueIndex + 1);
+                double variance = sum / count;
+                return Math.sqrt(variance);
+            }
+            return Double.NaN;
+        }
+
+        @Override
+        public String getName() {
+            return "stddev_pop";
+        }
+
+        // Chan et al. [CGL82; CGL83]
+        @Override
+        public void merge(MapValue destValue, MapValue srcValue) {
+            double srcMean = srcValue.getDouble(valueIndex);
+            double srcSum = srcValue.getDouble(valueIndex + 1);
+            long srcCount = srcValue.getLong(valueIndex + 2);
+
+            double destMean = destValue.getDouble(valueIndex);
+            double destSum = destValue.getDouble(valueIndex + 1);
+            long destCount = destValue.getLong(valueIndex + 2);
+
+            long mergedCount = srcCount + destCount;
+            double delta = destMean - srcMean;
+
+            // This is only valid when countA is much larger than countB.
+            // If both are large and similar sizes, delta is not scaled down.
+            // double mergedMean = srcMean + delta * ((double) destCount / mergedCount);
+
+            // So we use this instead:
+            double mergedMean = (srcCount * srcMean + destCount * destMean) / mergedCount;
```

Comment:
Why do we need to calculate the mean here? It's not used in `getDouble()`, only sum and count are used.
If you don't need the mean here, there is also no point calculating it in the base class for this implementation.